DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -95,7 +95,7 @@ public function process(ContainerBuilder $container)
 
                 $groupDefaults[$resolvedGroupName]['items'][] = array(
                     'admin'        => $id,
-                    'label'        => '',
+                    'label'        => !empty($attributes['label']) ? $attributes['label'] : '',
```

Comment:
the groups would not have the label set otherwise. https://sonata-project.org/bundles/admin/2-3/doc/reference/getting_started.html#step-3-create-an-admin-service

-> and further down I'll try to sort the entries by this label